views/instanceObservationTemplate.ejs

```diff
@@ -6,6 +6,7 @@
 
 <body>
   <% var orderData= data.orderData; %>
+  <% var matrixFormatedData= data.matrixFormatedData; %>
```

Comment:
@borkarsaish65 Adding points we discussed over the call

1. The box sizing in which the answer comes in
2. Font size of the questions
3. Font colour of who is answering the question (in app it is brown color)
4. indentation  of answers
5. Space between answers if there are multiple of them
6. Space between questions